exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Queries.kt

```diff
@@ -499,6 +499,26 @@ fun <T : Table> T.updateReturning(
     return ReturningStatement(this, returning, update)
 }
 
+fun <T : Table, E> T.batchUpdate(
+    data: Iterable<E>,
+    limit: Int? = null,
+    where: (SqlExpressionBuilder.() -> Op<Boolean>)? = null,
```

Comment:
Am I right that every statement may have own `where` condition?

For example:
```sql
update t set v = "v1" where id = 1
update t set v = "v2" where id = 2
...
```
Will it be reflected with this API? I just wonder because `where` arg is one for the whole statement, and the parameters for it should be taken somehow from the `data` parameter.